File: Python/Arrays/08.IntersectionOfArrays.py
from array import *

def intersetArray(arr, brr):

    for i in range (len(arr)):
        for j in range (len(brr)):
            if arr[i] == brr[j]:
                print(arr[i], end=' ')
                break


arr1 = array('i', [2, 3, 4, 6, 1, 5, 7])
arr2 = array('i', [3, 6, 5, 4, 3, 0, 1])

intersetArray(arr1, arr2)

File: CPP/Arrays/08.IntersectionOfArrays.cpp
#include <bits/stdc++.h>
using namespace std;


void intersectArr(vector<int> arr, vector<int> brr){
    
    for (int i =0; i< arr.size(); i++){
        for (int j = 0; j< brr.size(); j++){
            if (arr[i] == brr[j]){
                cout<<arr[i]<<" ";
            }
        }
    }
}


int main()
{
    vector<int> arr = {1, 3, 4, 6, 9, 7};
    vector<int> brr = {3, 5, 6, 4, 2};
    
    intersectArr(arr, brr);

    return 0;
}
